README.md

```diff
@@ -1,11 +1,22 @@
-# Getting Started
+# Getting Started with Jekyll
 1. install Ruby
 1. clone this repository
 1. run `bundle install`
 1. edit the source files
 1. run `bundle exec jekyll serve` to test changes on http://localhost:4000
 1. push changes to github to publish them
 
+# Contribute
+If you have an idea for a blog post or want to write a post to one of the existing ideas in the issue tracker,
+feel free to get in touch. Here's how we work:
+* For each month, we create a task board (see Projects Tab) that contains the blog posts and other tasks that we want to finish within that month.
+* We meet once a month to sort tasks into the upcoming months' boards and review the tasks of the previous month.
+* For each task, an issue in the issue tracker is created. Always include the issue number in your commits as described here: https://help.github.com/articles/closing-issues-via-commit-messages/
+* All tasks are proposed to the team as pull requests (PRs) which are reviewed and then merged. The repo is not modified directly.
+* If you have proposed a PR, put the ticket you were working on into the "Review" column of the task board.
+* If you have reviewed a PR and want changes, move the ticket back into the "In Progress" column.
```

Comment:
Also describe the way of creating a review via the `Files changed`  tab and how to approve / request changes.